src/uniprotkb/config/evidenceCodes.ts

```diff
@@ -262,15 +257,20 @@ export const ecoCodeToData = {
          */
         return 'Automatic assertion according to sequence analysis';
       }
-      if (isProtNLMEvidence(evidences)) {
-        return 'Automatic assertion from deep learning method';
-      }
       return 'Automatic assertion according to rules';
     },
     evidenceTagContentHeadingForGO:
       'Automatically inferred from sequence model',
     evidenceTagLabel: () => labels.AA,
   },
+  [ecoCode.PNLM]: {
+    manual: false,
+    evidenceTagContentHeading: () =>
+      'Automatic assertion from deep learning method',
```

Comment:
We should probably be using the wording from that will be associated with the ECO code, see forthcoming changes text by Vishal https://www.uniprot.org/release-notes/forthcoming-changes

Reply:
I see, I will change it